### PR TITLE
CASSANDRA-15046 Added CQLSH history command

### PR DESCRIPTION
**Pull request by bschoening**
Thanks for sending a pull request! Here are some tips if you're new here:
 
 * Ensure you have added or run the [appropriate tests](https://cassandra.apache.org/_/development/testing.html) for your PR.
 * Be sure to keep the PR description updated to reflect all changes.
 * Write your PR title to summarize what this PR proposes.
 * If possible, provide a concise example to reproduce the issue for a faster review.
 * Read our [contributor guidelines](https://cassandra.apache.org/_/development/index.html)
 * If you're making a documentation change, see our [guide to documentation contribution](https://cassandra.apache.org/_/development/documentation.html)
 
Commit messages should follow the following format:

```
<One sentence description, usually Jira title or CHANGES.txt summary>

<Optional lengthier description (context on patch)>

patch by <Authors>; reviewed by <Reviewers> for CASSANDRA-#####

Co-authored-by: Name1 <email1>
Co-authored-by: Name2 <email2>

```

The [Cassandra Jira](https://issues.apache.org/jira/projects/CASSANDRA/issues/)

### CODE REVIEW

File: doc/modules/cassandra/pages/managing/tools/cqlsh.adoc [hunk 1]
```diff
@@ -266,6 +266,18 @@ Gives information about cqlsh commands. To see available topics, enter
 `HELP <topic>`. Also see the `--browser` argument for controlling what
 browser is used to display help.
 
+=== `HISTORY`
+
+Prints to the screen the last n cqlsh commands executed on the server.
+The number of lines defaults to 50 if not specified
```
Comment (smiklosovic) [inline doc/modules/cassandra/pages/managing/tools/cqlsh.adoc]: add dot at the end of the line. "the last n" can be like "the last `n`"

File: pylib/cqlshlib/cqlshmain.py [hunk 2]
```diff
@@ -1835,6 +1835,31 @@ def do_help(self, parsed):
...
+        n = parsed.get_binding('n')
+        if (n is not None):
+             val = int(n)
+             self.max_history_length_shown = val
+
+        for index in range(history_length, history_length-self.max_history_length_shown, -1):
+            history_item = readline.get_history_item(index)
+            if history_item:
+                recent_history += history_item +"\n"
+        print("Recent Commands:\n\n" + recent_history)
```
Comment (smiklosovic) [inline pylib/cqlshlib/cqlshmain.py]: I am not sure we need to have "Recent Commands:\n\n". History on *nix does not print it either and it is pretty much self evident that we are printing history here.

Comment (bschoening) [reply pylib/cqlshlib/cqlshmain.py]: I agree, I'll remove it

File: doc/modules/cassandra/pages/managing/tools/cqlsh.adoc (hunk 1 above)
Comment (bschoening) [reply doc/modules/cassandra/pages/managing/tools/cqlsh.adoc]: fixed

**Pull request closed without merging**